src/openforms/forms/tests/test_api_formdefinition.py

```diff
@@ -156,6 +156,35 @@ def test_update(self):
         self.assertEqual(True, definition.login_required)
         self.assertIn({"label": "New field"}, definition.configuration["components"])
 
+    def test_update_is_reusable_unsuccessful_with_multiple_forms(self):
+        user = StaffUserFactory.create(user_permissions=["change_form"])
+        self.client.force_authenticate(user=user)
+
+        step = FormStepFactory.create(
+            form__internal_name="InternalForm",
+            form_definition__internal_name="InternalDef",
+            order=1,
+        )
+        FormStepFactory.create(
+            form_definition=step.form_definition,
+        )
```

Comment:
this is an impossible situation - the form definition is used in two forms but not marked as `reusable`.

**Mistakes**

Your test setup is wrong to begin with.